### ISSUE TITLE
Fake vehicle support

### ISSUE THREAD

**Issue by KindaChaotic69** [gta-chaos-mod/ChaosModV#2616]
Fake vehicle support (replace "Change current vehicle", like when fake death it shows the fake first, and after the real one).

**Effect Description**
Change the current vehicle for a fast, flying one, tanky, one cool to show from gta online, etc. But instead of that it is just a crappy bycicle, airport car, golf kart, and of course (with a pretty low chance) inside the train, aka softlocked.

**Comment by turg-the-sad-cat** [comment]
They don’t want softlocking effects

**Comment by kolyaventuri** [comment]
We do want to avoid softlocks, but that being said the idea of an effect where you get the appearance of a super nice car, but it has the stats of a golf cart sounds hilarious. Annoying, but easy enough to get out of

**Comment by KindaChaotic69** [comment]
yeah i kinda regreted about the softlock thing, but just don't include the train and i think it might work out with very funny situations

**Comment by KindaChaotic69** [comment]
> They don’t want softlocking effects

Delete your comment i edited the effect idea and deleted the softlock part, i don't want to confuse the creators xd. Anyway thank you for yor feedback! :)

**Comment by MoneyWasted** [comment]
> We do want to avoid softlocks, but that being said the idea of an effect where you get the appearance of a super nice car, but it has the stats of a golf cart sounds hilarious. Annoying, but easy enough to get out of

Verifying this request because of this content, not the original.
